Assets/Scripts/Interaction/Interact.cs

```diff
@@ -44,10 +44,9 @@ void CheckIfInteracting()
         {
             if (hit.collider.TryGetComponent(out Interactable interactable))
             {
-                interactable.Interact();
+                interactable.Interact(gameObject.transform);
```

Comment:
I made changes in my last PR to Interact. I can add the passing in of transform and if you get merge conflict when pulling take my files for this or discard changes.

Reply:
Noted